test/e2e/windows/service.go

```diff
@@ -56,16 +56,26 @@ var _ = SIGDescribe("Services", func() {
 		nodePort := int(svc.Spec.Ports[0].NodePort)
 
 		ginkgo.By("creating Pod to be part of service " + serviceName)
-		_, err = jig.Run(nil)
+		// tweak the Jig to use windows...
+		windowsTweak := func(rc *v1.ReplicationController) {
```

Comment:
nit: can we call this windowsNodeSelector or something like that?

Reply:
sure ? whatever is fine, did you wanna use that fancy github `insert a suggestion` button and ill just merge your commit as part of this?  or i can push one ?

Reply:
( i actually like windowsTweak bc, literally, the Jig names the input variable `tweak`) so would have a +.00001 preference to keep it named as is just for readability

Reply:
pushed a rename , `windowsNodeSelectorTweak` which probably satisfies all of our needs :)

Reply:
cool TIL the input variable is called tweak :-)